src/js/providers/html5.js

```diff
@@ -329,6 +329,10 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
     }
 
     function _play() {
+        const resumeLive = _videotag.duration === Infinity && _videotag.paused && _videotag.played && _videotag.played.length;
```

Comment:
Do you think we need to account for the case where the videotag is paused because it's stalling? The change would be (`_videotag.paused && _videotag.readyState > 1`)

Reply:
I don't see why. If the stream has played, you pause it, and then resume it, we want to restart the stream. Why do that if the stream never played? Let it finish buffering.